draft-ietf-rats-reference-interaction-models.md

```diff
@@ -407,7 +406,7 @@ For example, when performing a boot integrity evaluation, a Verifier may only re
 With the Handle, the Attestation Key IDs, and the Collected Claims, the Attester produces signed Evidence. That is, it digitally signs the Handle and the Collected Claims with a cryptographic secret identified by the Attestation Key ID. This is done once per Attesting Environment which is identified by the particular Attestation Key ID. The Attester communicates the signed Evidence as well as all accompanying Event Logs back to the Verifier.
 
 The Claims, the Handle, and the Attester Identity information (i.e., the Authentication Secret) MUST be cryptographically bound to the signature of Evidence. These MAY be presented obfuscated, encrypted, or cryptographically blinded.
-For further reference see section {{security-and-privacy-considerations}}.
+For further reference see Section {{security-and-privacy-considerations}}.
```

Comment:
-= Section (otherwise it’ll render as “Section Section 9”)
+= comma

```suggestion
For further reference, see {{security-and-privacy-considerations}}.
```